docs/changelog/1_0_a5.rst

```diff
@@ -0,0 +1,60 @@
+===========
+1.0 Alpha 5
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 5 "Luhman 16".
+
+
+EdgeQL
+======
+
+* Implement casts between JSON and enums, arrays and tuples
+  (:eql:gh:`#251`).
+* Allow constraints on tuple types (:eql:gh:`#1576`).
+* Allow constraints directly on object types in SDL (:eql:gh:`#1164`)
+* Disallow using :eql:constraint:`exclusive` on scalar types
+  (:eql:gh:`#1575`).
+* Proper implementation of ``SET/DROP OWNED``.
+* Fix issues with some ``FOR`` statements (:eql:gh:`#1594`).
+* Use fully-qualified names to disambiguate the expressions produced
+  by :eql:stmt:`DESCRIBE` (:eql:gh:`#1254`).
+* Initial implementation of ``INSERT ... ON CONFLICT ... ELSE``
+  (:eql:gh:`#1369`)
+* Implementation of more of the features of the new migration syntax
+  (`RFC 1000 <migrations_>`_).
+
+
+GraphQL
+=======
+
+* Allow several mutation operations in a single mutation query
+  (:eql:gh:`#1569`).
+* Reflect nested aliased types (:eql:gh:`#722`).
+* Enable sorting on non-trivial path (:eql:gh:`#1642`).
+* Add an ``exists`` filter operation (:eql:gh:`#1655`).
+
+
+CLI
+===
+
+* Reworked auth setup via ``edgedb server init`` (`#91
+  <https://github.com/edgedb/edgedb-cli/issues/91>`_).
+* Initial support for the migrations CLI.
+* Add ``edgedb server status --all`` command to list all instances.
+
+
+Misc
+====
+
+* Add support of connecting to instance by a name (`#112
+  <https://github.com/edgedb/edgedb-python/pull/113>`_).
+* Update the `edgedb-js <https://github.com/edgedb/edgedb-js>`_ driver
+  to v0.10.0.
```

Comment:
Looks like we are 0.9.0 on edgedb-js:

https://www.npmjs.com/package/edgedb

Reply:
Yes, 0.9.0 is the latest for edgedb-js, and 0.10.0 is the latest for edgedb-python.

Reply:
Fixed.